handbook/product/roles/product_designer.md

```diff
@@ -57,10 +59,10 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
    - [30 min] [HM intro call](./interviews/hm_intro_call.md) with [Christina Forney](../../../company/team/index.md#christina-forney-she-her) (VP Product)
 1. [1 hr] **[Portfolio review](./interviews/pd_portfolio_review.md)**: you to present your past work.
```

Comment:
```suggestion
1. [1 hr] **[Portfolio review](./interviews/pd_portfolio_review.md)**: you to present impactful projects from your portfolio, including a case study detailing a specific project
```

Reply:
@rrhyne I think that we want the candidate to present only one project, not 'impactful projects' wdyt?

Reply:
I agree with setting the expectation clearly, and I'd much prefer a deep dig into one project than a surface-level look at several.